src/main/java/org/broadinstitute/dsm/db/ViewFilter.java

```diff
@@ -261,10 +261,20 @@ public static void addQueryCondition(@NonNull Map<String, String> queryCondition
         }
         else {
             String queryCondition = "";
-            if (queryConditions.containsKey("ES")) {
-                queryCondition = queryConditions.get("ES");
+            if (filter != null && filter.getParentName() != null &&
```

Comment:
I think extracting this long if condition into method with meaningful name would be useful.